src/Root_TEST.cc

```diff
@@ -270,3 +295,48 @@ TEST(DOMRoot, FrameSemanticsOnMove)
     testFrame1(root2);
   }
 }
+
+/////////////////////////////////////////////////
+TEST(DOMRoot, AddWorld)
+{
+  sdf::Root root;
+  EXPECT_EQ(0u, root.WorldCount());
+
+  sdf::World world;
+  world.SetName("world1");
+  sdf::Errors errors = root.AddWorld(world);
+  EXPECT_TRUE(errors.empty());
+  EXPECT_EQ(1u, root.WorldCount());
+  EXPECT_FALSE(root.AddWorld(world).empty());
```

Comment:
nit: I think the `ASSERT_FALSE` is sufficient.

Reply:
Removed.